Find a functionally equivalent code:android.net.wifi.WifiInfo.getHiddenSSID:COMMENT
Method Modifier: public      
Comment:/**
 * @return {@code true} if this network does not broadcast its SSID, so an
 * SSID-specific probe request must be used for scans.
 */

@@ -1,3 +1,5 @@
 {
-    return mHiddenSSID;
+    if (mWifiSsid == null)
+        return false;
+    return mWifiSsid.isHidden();
 }

